components/Paper/Upload/api/getSuggestedTags.ts

```diff
@@ -0,0 +1,28 @@
+import API from "../../../../config/api";
+import { Helpers } from "@quantfive/js-web-config";
+
+type Payload = {
+  onError?: Function | null;
+  onSuccess: Function;
+  searchString: string;
+};
+
+export function getSuggestedTags({ onError, onSuccess, searchString }: Payload): void {
+  fetch(API.TAG({ limit: 1000, search: searchString }), API.GET_CONFIG())
+    .then(Helpers.checkStatus)
+    .then(Helpers.parseJSON)
+    .then((resp: any): void => {
+      const tags = resp.tags
+        .map((tag, _index) => {
```

Comment:
I know you're following the pattern of getSuggestedHubs here which I happened to write based on some legacy code for convenience. :/.
I was wondering if we can migrate this to BE & we debounce the request instead of fetching all of the tags all at once.